apps/design/shared/src/env.d.ts

```diff
@@ -4,3 +4,8 @@ declare namespace NodeJS {
     readonly CI?: string;
   }
 }
+
+// Both Node and browser support console.log
+declare module console {
+  export function log(message?: any, ...optionalParams: any[]): void;
+}
```

Comment:
Because apps/design/shared needs to be used in both frontend and backend runtimes, the TS config can't include settings for a Node or browser environment. Those settings (which are in most of our other TS configs) tell TS that `console.log` exists. Without that setting, I was getting TS errors every time I tried to debug using `console.log`. This was a workaround that I came up with to tell TS that `console.log` exists.